dmscripts/bulk_upload_documents.py

```diff
@@ -1,11 +1,10 @@
 import os
 import re
-import sys
-sys.path.insert(0, '.')
-if sys.version_info > (3, 0):
-    import csv
-else:
+try:
     import unicodecsv as csv
+except ImportError:
+    import csv
```

Comment:
Do we still support python2 for scripts? I thought we changed over.

Although I accept removing this could be beyond the flake8 scope.

Reply:
Some of the stuff in scripts is still running in a py2 env in on jenkins:
https://ci.marketplace.team/job/upload-dos-opportunities-email-list-production/100/consoleFull
```
  File "/data/jenkins/jobs/upload-dos-opportunities-email-list-production/workspace/venv/lib/python2.7/site-packages/mailchimp3/mailchimpclient.py", line 95, in _get
```